project/urls.py

```diff
@@ -8,7 +9,7 @@
     path("events/", include("techcity.events.urls")),
     path("groups/", include("techcity.groups.urls")),
     path("api/", include("techcity.api.urls")),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Using this is probably less efficient. We don't have an Nginx reverse proxy in place, so I think this is an ok option for now. In the future, if the performance is a problem, I may explore serving media files through WhiteNoise. Doing that would at least add long cache headers to requests.